src/core/public/chrome/ui/loading_indicator.test.tsx

```diff
@@ -32,7 +32,10 @@ describe('kbnLoadingIndicator', () => {
 
   it('is visible when loadingCount is > 0', () => {
     const wrapper = shallow(<LoadingIndicator loadingCount$={new BehaviorSubject(1)} />);
-    expect(wrapper.prop('data-test-subj')).toBe('globalLoadingIndicator');
+    // Pause the check beyond the 250ms delay that it has
+    setTimeout(() => {
+      expect(wrapper.prop('data-test-subj')).toBe('globalLoadingIndicator');
+    }, 300);
```

Comment:
optional: It could be a source of flakiness on CI. We might implement a function polling the element presence (as in FTR [TestSubjects.exists](https://github.com/restrry/kibana/blob/7c6a40ea34872a1c08ac5c1e01c19c99e1742030/test/functional/services/common/test_subjects.ts#L48) or remove the time factor in tests https://jestjs.io/docs/en/timer-mocks#run-all-timers